gui/src-tauri/src/state.rs

```diff
@@ -73,7 +75,11 @@ impl WindowState {
 			window.unmaximize()?;
 		}
 
-		window.set_size(LogicalSize::new(self.width, self.height))?;
+		if self.width > util::MIN_WINDOW_SIZE_WIDTH
+			&& self.height > util::MIN_WINDOW_SIZE_HEIGHT
+		{
+			window.set_size(LogicalSize::new(self.width, self.height))?;
+		}
```

Comment:
Why not just use `std::cmp::max()`?

Reply:
the idea is to not set size if somehow it saved an incorrect one

Reply:
nvm, you're right